tests/Predis/Command/Redis/TimeSeries/TSALTER_Test.php

```diff
@@ -89,6 +104,36 @@ public function testAlterUpdatesExistingTimeSeriesConfiguration(): void
         );
     }
 
+    /**
+     * @group connected
+     * @group relay-resp3
+     * @return void
+     * @requiresRedisTimeSeriesVersion >= 1.12.0
+     */
+    public function testAlterUpdatesExistingTimeSeriesWithIgnoreOption(): void
+    {
+        $redis = $this->getClient();
+
+        $arguments = (new CreateArguments())
+            ->retentionMsecs(60000)
+            ->duplicatePolicy(CommonArguments::POLICY_MAX)
+            ->labels('sensor_id', 2, 'area_id', 32);
+
+        $this->assertEquals(
+            'OK',
+            $redis->tscreate('temperature:2:32', $arguments)
+        );
+
+        $alterArguments = (new AlterArguments())
+            ->retentionMsecs(10000000)
+            ->ignore(10, 10);
+
+        $this->assertEquals(
+            'OK',
+            $redis->tsalter('temperature:2:32', $alterArguments)
+        );
```

Comment:
Consider trying to add some values close to each other, before/after the alter, and make sure after alter they are not accepted?